onnxruntime/test/python/node/depthwise_conv2d.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np  # type: ignore
+import tensorflow as tf
+import tf2onnx
+from onnx.backend.test.case.base import Base
```

Comment:
I would avoid importing a class or a function from a module.

Import only the module instead to keep the name space clean and provide readers with more context on where its members come from. For example, readers don’t need to backtrack to see `sleep` is a function from `time`, as opposed to a function defined in the file. https://google.github.io/styleguide/pyguide.html#22-imports